bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -9,7 +11,11 @@ $(LINT_DIR)/bsg_tag_boot_rom.v: $(TB_PATH)/$(TB)/bsg_tag_boot.tr
 $(LINT_DIR)/flist.vcs:
 	@grep -v -e "^\#" $(SYN_PATH)/flist.vcs          > $@
 	@grep -v -e "^\#" $(TB_PATH)/$(TB)/flist.vcs    >> $@
-	@echo wrapper.sv                                >> $@
+ifeq ($(SYNTH_P), 0)
+	@echo wrapper.sv                              >> $@
```

Comment:
Don't change the flist here, edit the wrapper.sv target. Otherwise it'll be confusing with two wrappers in the build directory